### DISCUSSION TITLE
Income brackets in vaccination data

### DISCUSSION THREAD

**Discussion by Linus-Foley** [owid/covid-19-data#2291] [Q&A] [answered]
Hello, I've noticed that vaccinations.csv data contains income brackets as locations (like High, Low, Upper Middle). I was wondering, how are these brackets determined? Does it use GNI per capita, [like the World Bank does here?](https://datatopics.worldbank.org/world-development-indicators/stories/the-classification-of-countries-by-income.html)

Thanks, 
Linus

**Comment by edomt** [comment, marked as answer]
Yes, our income groups are indeed the exact ones used by the World Bank. You can find the exact list [in this file](https://github.com/owid/covid-19-data/blob/master/scripts/input/wb/income_groups.csv).
